Assets/Scripts/FloatingOrigin/Origin.cs

```diff
@@ -10,6 +10,8 @@ namespace Netherlands3D.Twin.FloatingOrigin
     public class Origin : MonoBehaviour, IHasCoordinate
     {
         public Coordinate Coordinate { get; set; }
+
+        public static Origin current;
```

Comment:
Just a naming thing, can you call this "Instance" as it looks like a more common way of naming a Singleton field. Current also implies there can be multiple and this is the current one, while there is one Origin.

Reply:
⚠️ And instead of using a field, please use a property that can populate itself. That way, you will always have a value available and do not have issues with order of awake's and starts

Reply:
I used EventSystem.current as a reference for this, just like our own ProjectData.current. Felt consistent.
To improve it even further we could avoid having to do null checks by setting it in the ConfigLoader scene, so we can assume it is there in all other scripts running in main.

Reply:
Will be Singleton